internal/models/server.go

```diff
@@ -44,7 +44,7 @@ var (
 	ErrWireguardPublicKeyEmpty = errors.New("wireguard public key field is empty")
 )
 
-func (s *Server) HasMinimumInformation() (err error) {
+func (s *Server) HasMinimumInformation(providerWireguardKeyUnknown bool) (err error) {
```

Comment:
instead check the `Hostname` field and determine if it's a PIA server from this? So there is no need for the extra argument 🤔